content/workload-reduction-toolkit/address-workload-issues/curriculum-planning-and-delivery/develop-a-culture-of-support-and-staff-development.md

```diff
@@ -0,0 +1,144 @@
+
+### Use tools to measure impact 
+
+Alongside assessment data, as part of our goal to ‘Make the difference’, we measure our impact against our stated aims in a variety of other ways including surveys and measurable targets. These are recorded and analysed in our Impact Tracker.  
+
+## Celebrating achievements 
+
+### Share best practice and awards  
+
+We have a variety of awards and accreditations, all of which provide opportunities for reflection and analysis, helped by the numerous schools who visit us to find out how we do things. Their reflections help us to reflect and move forward. 
+
+### Provide coaching for all staff  
+
+Our coaching approach provides opportunities for us to be reflective and identify our own areas for improvement. This is done through peer groups and networks. All senior leaders meet regularly with the Head Teacher to be coached and support their further development. The Head Teacher meets regularly with the academy trust and an external coach to support his further development. 
+
+## Developing ourselves and supporting each other 
+
+Once we understand where to improve, we need to work together to achieve this. We use a range of methods to achieve this: 
+
+### Provide formal and informal feedback  
+
+Feedback will always be positive and supportive. It will provide high challenge to encourage and inspire people to be their best, alongside high levels of support.  
+
+### Individualised continuing professional development (CPD) for all staff 
+
+Individualised CPD will be linked to the areas identified in Performance Development. This is personalised by individuals to develop the areas they want to develop in.  
+
+### Provide whole school CPD   
+
+Whole school CPD is delivered through INSETs and staff training on Mondays after school. INSETs are used as opportunities for team building and to deliver information on new initiatives along with statutory updates. Staff training is carefully planned to allow sufficient time and space for reflection and implementation. Everyone is welcome to attend this.  
+
+### Encourage collaborative working  
+
+Everyone at Kensington works in teams. Being part of a team supports the development of ideas, the sharing of workload, the wellbeing of us all, and the overall improvement of the school.  
+
+### Utilise coaching from within the school and from across our Trust 
+
+Utilising and contributing to expertise and knowledge across the schools to support one another, coupled with more formal support and challenge sessions for the headteacher from the Trust CEO to aid the on-going development of the school. 
+
+### Carry out whole school peer reviews  
+
+We carry out a whole school peer review every 2 years. This peer review draws more extensively on the broad expertise from across the Trust and external partners to challenge the school as part of the ongoing development process. 
```

Comment:
This sentence is quite long and could be refined - "We carry out a whole school peer review every 2 years. This review draws extensively on the expertise from across the Trust and external partners, to challenge the school as part of the ongoing development process".